### PR TITLE
Restore the template in working state

### PR DESCRIPTION
**Pull request by dereckson**
The November 2017 commits starting with 55b5b0b1 used the template repository
to inject code for the a customer project.

The commit ff93bd892 hided this situation fixing the README.

This change fixes this situation restoring genuine Foundation settings,
the former style guide and content, without the customer customisation.

Documentation and supports links are updated for Foundation 6.

### CODE REVIEW

**Comment by ncoden** [comment]
Hey @rafibomb, what do you think of this ?

File: readme.md [hunk 1]
```diff
@@ -12,69 +12,59 @@ This is the official ZURB Template for use with [Foundation for Sites](http://fo
...
 
 ## Installation
 
 To use this template, your computer needs:
 
-- [NodeJS](https://nodejs.org/en/) (Up to Version 8 only, Tested with 8.12.0)
+- [NodeJS](https://nodejs.org/en/) (Version 6 or greater recommended, tested with 6.11.4 and 8.12.0)
 - [Git](https://git-scm.com/)
 
-### Setup
```
Comment (rafibomb) [inline readme.md:25]: this section should also be in here. This shows the manual setup way.

File: readme.md [hunk 2]
```diff
@@ -12,69 +12,59 @@ This is the official ZURB Template for use with [Foundation for Sites](http://fo
...
-Then open the folder in your command line, and install the needed dependencies:
+Install the Foundation CLI with this command:
 
 ```bash
-cd projectName
-yarn
+npm install foundation-cli --global
 ```
 
-Now `cd` to your project name and run `npm start` or `foundation watch` to run Gulp. Your finished site will be created in a folder called `dist`, viewable at this URL:
```
Comment (rafibomb) [inline readme.md]: should say 
Now `cd` to your project name and run `yarn start`  to run Gulp. While developing, you can view your project at http://localhost:8000. Your finished site will be created in a folder called `dist`.

**Comment by rafibomb** [comment]
@dereckson @ncoden Yes, this should definitely happen and thanks for the PR!

Those changes were meant to live on the branches `zurb-webapp` and `zurb-marketing`

**Comment by ncoden** [comment]
Thank you @rafibomb for the review. I did not know if this was the intention of ZURB to update the template that way.

File: readme.md (hunk 1 above)
Comment (ncoden) [reply readme.md:25]: Download and installation instructions are still there, moved to `### Manual Setup` below.

File: readme.md (hunk 2 above)
Comment (ncoden) [reply readme.md]: This is basically what is being said in `### Manual setup` below:

> Finally, run `yarn start` to run Gulp. Your finished site will be created in a folder called `dist`, viewable at this URL:
> ```
> http://localhost:8000
> ```

**Review by ncoden — APPROVED** [review]
Hi @dereckson 👋 

I resolved conflicts and added back `package-lock.json`.
LGTM 👍

**Comment by ncoden** [comment]
@rafibomb good to merge ?

**Review by rafibomb — APPROVED** [review]

**Comment by dereckson** [comment]
You're welcome. Thanks for the fast review.

**Pull request merged**